libraries/botframework-connector/src/auth/authenticationConstants.ts

```diff
@@ -149,4 +150,14 @@ export namespace AuthenticationConstants {
      * Service URL claim name. As used in Microsoft Bot Framework v3.1 auth.
      */
     export const ServiceUrlClaim = 'serviceurl';
+
+    /**
+     * AppId used for creating skill claims when there is no appId and password configured.
+     */
+    export const AnonymousSkillAppId = 'AnonymousSkill';
+
+    /**
+     * Indicates that bot identity is anonymous (no appId and password were provided).
+     */
+    export const AnonymousAuthType = 'anonymous';
```

Comment:
I don't see the same `ClaimsIdentity` constructor that takes a string as its second parameter, so at the moment this is unused.